package-lock.json

```diff
@@ -7391,28 +7391,28 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "",
+          "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
```

Comment:
Do we want to add all of these _resolved_ urls?

Reply:
This file is autogenerated by npm -- happens during minor version updates. It can be ignored.